assembl/static2/css/views/administration.scss

```diff
@@ -229,6 +234,10 @@
       }
     }
 
+    .legal-content-section {
+      padding: 20px 0;
+    }
```

Comment:
Do you really need a specific class like this? Can't you use a generic one or modify existing style to have a generic class that handle this padding? As a rule of thumb, think twice about adding a new classname.

Reply:
I agree, we could use the spacing classes from bootstrap like "py-1" or "py-2" but I have not seen it in the project, I thought it was for a specific reason.